eks-distro-base/tests/Dockerfile

```diff
@@ -28,6 +28,9 @@ CMD ["/bin/check-cgo"]
 
 FROM ${BASE_IMAGE} as check-iptables-legacy
 
+RUN ["update-alternatives", "--set", "iptables", "/usr/sbin/iptables-legacy"]
```

Comment:
this is a dockerfile we use to run some basic docker tests with these images. I changed the legacy target to explicitly set the alternative and removed it from the wrapper target.